tee-worker/service/src/main_impl.rs

```diff
@@ -98,7 +98,7 @@ pub(crate) fn main() {
 	// stdout from the service
 	#[cfg(feature = "production")]
```

Comment:
Should we keep the `production` feature? or should we do:
` #[cfg(not(feature = "development"))]`

Reply:
I actually prefer latter as `production` should be the default now.

Is there a better name for `development`? :D 

cc @kziemianek too

Reply:
`development` makes sense to me tbh :thu

Reply:
Agree, imho we should have only `development` feature.

Reply:
It makes sense, thank you guys! 👍🏼